bin/pull-latest-release-asset.sh

```diff
@@ -3,8 +3,20 @@
 repoName=$1
 assetName=$2
 
-latestAssetId=$(curl https://api.github.com/repos/hmcts/${repoName}/releases/latest \
-  | docker run --rm --interactive stedolan/jq ".assets[] | select(.name==\"${assetName}\") | .id")
+retries=0
+while [ -z "$latestRelease" ]
```

Comment:
We should include the second curl call in the retry loop, otherwise the build will fail after failing to pull the asset once. And we could check if `$assetName` file exists as the loop condition.